src/components/asset-list-item/AssetListItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { IconWarning } from '../..';
+import { SFlex } from '../../base-components/flex/styles/SFlex';
+import { IconCheckmarkOutlined } from '../../icons/IconCheckmarkOutlined';
+import { IconCross } from '../../icons/IconCross';
+import { SAssetListItem } from './styles/SAssetListItem';
+import { SAssetListItemDescription } from './styles/SAssetListItemDescription';
+import { SAssetListItemIconWrapper } from './styles/SAssetListItemIconWrapper';
+import { SAssetListItemTextWrapper } from './styles/SAssetListItemTextWrapper';
+import { SAssetListItemTitle } from './styles/SAssetListItemTitle';
+import { SImagePlaceholder } from './styles/SImagePlaceholder';
+import { SProgress } from './styles/SProgress';
+import { SProgressBackground } from './styles/SProgressBackground';
+import { SProgressBar } from './styles/SProgressBar';
+import { SProgressBarContainer } from './styles/SProgressBarContainer';
+
+export interface IAssetListItemProps {
+  title?: String | undefined,
+  description?: String | undefined,
+  icon?: React.ReactNode,
+  recent?: boolean,
+  error?: boolean,
+  uploading?: boolean,
+  progressValue: number,
+}
+
+export const AssetListItem: React.FC<IAssetListItemProps> = ({ ...props }) => {
+  console.log(props);
+  return (
+    <SAssetListItem {...props}>
+      <SFlex style={{ width: "100%" }}>
```

Comment:
@steiningermona you can extend this Styled component locally. Please avoid inline styling.

const SFlexListItem = styled(SFlex)``